packages/wrangler/src/autoconfig/c3-vendor/add-wrangler-gitignore.ts

```diff
@@ -113,3 +113,94 @@ export const addWranglerToGitIgnore = (projectPath: string) => {
 		)}`
 	);
 };
+
+export const addWranglerToAssetsIgnore = (projectPath: string) => {
```

Comment:
mh.... I don't think that we should add non-c3 logic in this `c3-vendor` directory, no? 🤔